Sources/Opus/Opus.Decoder.swift

```diff
@@ -5,9 +5,10 @@ public extension Opus {
 	class Decoder {
 		let format: AVAudioFormat
 		let decoder: OpaquePointer
+		let customFrameSize: Int32?
```

Comment:
This can probably be a non-optional int that defaults to `0`.

We’ll also need to add a `let customMode: OpaquePointer` here (see below).

Reply:
Actually no, this is a way to avoid having a duplicate un-needed pointer.
If you have a custom frame size, then your decoder is a custom one.

Reply:
There’s no function in the Opus library to extract the frame size used to initialize an `OpusCustomMode`, hence storing it here.

Reply:
I don't follow your point here; can you clarify?